samples/boards/nxp/adsp/rtxxx/amp_blinky/sysbuild.cmake

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
The only accepted copyright from NXP Legal is 

``Copyright 2025 NXP`` (remove (c))

Reply:
For the record:

```
grep "Copyright (c).*NXP" . -R | wc -l
489
```

plus a bunch of other variations as well :)